Makefile

```diff
@@ -1,26 +0,0 @@
-BINDIR = /usr/local/bin
```

Comment:
@pasuder pls restore the file as you committed it on an interim commit, I'd like folks to have it as a fallback in case of trouble with `setup.py` at least until git-machete is published to PyPI. Also pls mention that in Makefile as an second installation alternative... I guess this is gonna be the safer option :) sorry if you have the impression that I changed my mind on that! :)

Reply:
OK.